### ISSUE TITLE
 Issue with 'Docs' button on Home page (on Framer)

### ISSUE THREAD

**Issue by Naprila** [twentyhq/twenty#7661]
## Bug Description

On hover over Docs button, background css (light grey square shape) is not showing up while hovering over other buttons shows it

screenshot 1: (when cursor is on Docs button)
<img width="1256" alt="Docs-hover" src="https://github.com/user-attachments/assets/d4ed39ef-f8f9-47ef-843f-37c81f5b6394">

screenshot 2: (when cursor is on other button)
<img width="1263" alt="Releases-hover" src="https://github.com/user-attachments/assets/37e0f2a3-c56a-43d0-ac87-300dd8e5daeb">

## Expected behavior

Same background effect should be there as it is on hovering over other buttons like Story, Pricing, Releases.

## Technical inputs

- Probably fixable with CSS

**Comment by Ahmadmidlaj** [comment]
i would like to work on this

**Comment by MK884** [comment]
yes it works on developer page navigation but not on main landing page 

checkout => https://twenty.com/developers

![image](https://github.com/user-attachments/assets/c634b828-f713-4a0b-a49c-fe38d007e4a9)

Interesting issue, I would like to work on it.

**Comment by vamus092** [comment]
is the issue resolved

**Comment by Bonapara** [comment]
No, but I'll handle it since it concerns the Framer website!
